snapcraft/internal/repo/_base.py

```diff
@@ -217,6 +222,49 @@ def normalize(self, unpackdir):
         self._fix_xml_tools(unpackdir)
         self._fix_shebangs(unpackdir)
 
+    def read_source_package_attr(self, path: str) -> Optional[str]:
+        if sys.platform != "linux":
+            raise RuntimeError("xattr support only available for Linux")
+
+        # Extended attributes do not apply to symlinks.
+        if not os.path.islink(path):
+            return None
```

Comment:
lamest nit of all, but mind adding a newline after this return call so it these early guards are visually separated from the rest of the logic?

Reply:
will do